fastify.js

```diff
@@ -251,8 +251,16 @@ function build (options) {
   }
 
   function override (old, fn, opts) {
+    const prefix = opts.prefix
     if (fn[Symbol.for('skip-override')]) {
-      return old
+      if (!prefix) {
+        return old
+      }
+
+      const instance = Object.create(old)
+      instance._RoutePrefix = buildRoutePrefix(instance._RoutePrefix, opts)
+
+      return instance
```

Comment:
This is not what we want. In fact, we should not do this for plugins created via fastify-plugin. It's part of the deal: you are modifying your parent context.

Reply:
Agree, if you want to handle the prefix inside a plugin with `fastify-plugin` you should handle it by yourself, as we did in [`fastify-bankai`](https://github.com/fastify/fastify-bankai).

Reply:
Yeah, I'm not convinced too